spec/components/all_components_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe "All components" do
   it "doesn't use `html_safe`" do
-    files_with_html_safe = `grep -rni "html_safe" app/views/govuk_publishing_components/components`.lines
+    files_with_html_safe = `grep -rni "html_safe " app/views/govuk_publishing_components/components`.lines
```

Comment:
Why this change?

Reply:
Lol I hoped you wouldn't notice. It's to avoid this test triggering on the deprecation message in the component.